assets/js/base/hocs/with-browser-window-prop.js

```diff
@@ -0,0 +1,12 @@
+const w = window || {};
+
+const withBrowserWindowProp = ( prop, propMap ) => ( WrappedComponent ) => (
+	incomingProps
+) => {
+	const props = w.hasOwnProperty( prop )
+		? propMap( w[ prop ] || {}, incomingProps )
```

Comment:
Do you need the `w[ prop ] || {}` here?  Isn't there already a check for if the window has the property.  There are some window properties that are booleans, numbers, strings which could have a non-truthy value.  Would it be safe to default those to an empty object?

Reply:
Good catch! Fixed in 8c96426f5030bce34501383abe27a2f788101e94.